btax/front_end_util.py

```diff
@@ -44,7 +54,10 @@ def runner_json_tables(test_run=False,start_year=2016,iit_reform=None,**user_par
                     tables[k1][k2] = v2
         else:
             raise ValueError('Expected an "asset" or "industry" related table')
-    return add_summary_rows_and_breaklines(dict(tables), start_year, do_assertions=test_run)
+    return json.dumps({
+        'json_table': add_summary_rows_and_breaklines(dict(tables), start_year, do_assertions=test_run),
+        'dataframes': serialized_dataframes,
+        })
```

Comment:
@jdebacker I think this should be: 
```
+    return {
+        'json_table': add_summary_rows_and_breaklines(dict(tables), start_year, do_assertions=test_run),
+        'dataframes': serialized_dataframes,
+        }) 
```
Before, a Python dictionary was passed back to webapp.  The new code returns a string but the webapp expects a dictionary.  Making this change resolves one of the webapp compatibility errors.